docs/changelog/4.2.rst

```diff
@@ -0,0 +1,1146 @@
+
+    Handle ANY queries with Lua records.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 6869
+
+    Remove out-of-zone-additional-processing setting.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 7217
+
+    geoip: properly delete libGeoIP return values.
+
+  .. change::
+    :tags: Improvements, IXFRDist
+    :pullreq: 7182
+
+    Add metrics to ixfrdist.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 7067
+
+    SOA-check: reject nxdomain response and check label of RR against qname.
+
+  .. change::
+    :tags: Improvements, DNSSEC
+    :pullreq: 6958
+
+    Improve RSA key warnings
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 7201
+
+    Fix carbon-instance / carbon-namespace inconsistencies.
+
+  .. change::
+    :tags: <nil>
+    :pullreq: 7196
+    :tickets: 7195
+
+    geoipbackend: Allow empty content for ENT record.
+
+  .. change::
+    :tags: Tools, DNSSEC
+    :pullreq: 7187
+
+    pdnsutil.1 & settings: Add Ed25519 and Ed448, document ECC keysizes, remove old algos
+
+  .. change::
+    :tags: Bug Fixes, API
+    :pullreq: 6871
+
+    Check DNSNames that should be hostnames.
+
+  .. change::
+    :tags: <nil>
+    :pullreq: 6959
+    :tickets: 2362, 6951
+
+    Add namespace and instance variable to carbon key.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 7111
+
+    Rename skipLabel function to skipDomainName.
+
+  .. change::
+    :tags: Bug Fixes, Packages
+    :pullreq: 7145
+
+    m4: Fix lua.hpp check to actually work
+
+  .. change::
+    :tags: Bug Fixes, Packages
+    :pullreq: 7134
+
+    Fix up the BIND config files on upgrade
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 7024
+
+    geoipbackend: Handle read error for config file
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 7074
+
+    Print the content of pdns.log when the regression tests fail.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 7069
+
+    Use unique pointers in the OpenSSL signer.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 7057
+    :tickets: 5512
+
+    ``scopeMask`` in the ``SOAData`` structure is unused after #5512.
+
+  .. change::
+    :tags: API, Removed Features
+    :pullreq: 7025
+
+    Remove ``api-logfile`` flag and grep API endpoint.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 6962
+
+    Store NetmaskTree nodes in a set for faster removal
+
+  .. change::
+    :tags: New Features
+    :pullreq: 6969
+
+    A clean re-do of an earlier PR to add the glorious log-log histograms
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 7017
+
+    Make sure we escape 127 in TXT records.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 4598
+
+    Add support for ``NONE`` SOA-EDIT kind
+
+  .. change::
+    :tags: Tools
+    :pullreq: 6997
+
+    Name threads in the programs.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 6727
+
+    ALIAS: Respond SERVFAIL on non-NOERRORs from resolver.
+
+  .. change::
+    :tags: New Features
```

Comment:
4.1 did support ed448/25519, just not with openssl, so not sure this is a new feature